### PR TITLE
fix

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -55,7 +55,7 @@ ram.runtime = "50M"
     main.url = "/"
 
     [resources.apt]
-    packages = "postgresql, apt-transport-https, postgresql-contrib, php8.2-zip, php8.2-pgsql, php8.2-xml, php8.2-intl, php8.2-gd, php8.2-curl, php8.2-bcmath, php8.2-opcache"
+    packages = "postgresql, apt-transport-https, postgresql-contrib, php7.4-zip, php7.4-pgsql, php7.4-xml, php7.4-intl, php7.4-gd, php7.4-curl, php7.4-bcmath, php7.4-opcache"
```
Comment (SimonMellerin) [inline manifest.toml]: v2.0.x of æneria requires php8.2

File: scripts/install [hunk 2]
```diff
@@ -24,8 +25,8 @@ ynh_script_progression --message="Setting up source files..." --weight=1
 # Download, check integrity, uncompress and patch the source from app.src
 ynh_setup_source --dest_dir="$install_dir"
 
+chmod -R o-rwx "$install_dir"
```
Comment (SimonMellerin) [inline scripts/install:28]: this permission is not enough that's why i put back 

`chmod -R 750 "$install_dir"`

File: scripts/upgrade [hunk 3]
```diff
@@ -67,8 +67,8 @@ then
   ynh_secure_remove --file="$tmpdir"
 fi
 
+chmod -R o-rwx "$install_dir"
```
Comment (SimonMellerin) [inline scripts/upgrade:70]: this permission is not enough that's why i put back 

`chmod -R 750 "$install_dir"`

**Review by SimonMellerin — CHANGES_REQUESTED** [review]
Thanks for your fixes and review.

File: scripts/upgrade (hunk 3 above)
Comment (ericgaspar) [reply scripts/upgrade:70]: `chmod -R 750 "$install_dir"` should be automatically applied to the installation directory when it is created

Comment (SimonMellerin) [reply scripts/upgrade:70]: ok, but then why do you add this `chmod -R o-rwx "$install_dir"` ?

Comment (ericgaspar) [reply scripts/upgrade:70]: I notice on other packages with permission issues that adding `chmod -R o-rwx "$install_dir"` solves the issue...

Comment (SimonMellerin) [reply scripts/upgrade:70]: ok, well for æneria, it seems to break the app :/

Nginx can no longer reads public static assets with this permission

Comment (ericgaspar) [reply scripts/upgrade:70]: oh well, #46 is successful 🏋🏼. closing this one

**Pull request closed without merging**